### ISSUE TITLE
Error run_camsa.py

### ISSUE THREAD

**Issue by navajeet123** [compbiol/CAMSA#17]
Hi,

I am getting an error while executing run_camsa.py
Please suggest!
=============
2020-04-12 20:48:48,149 - CAMSA.main      - INFO    - Starting the analysis
2020-04-12 20:48:48,149 - CAMSA.main      - INFO    - Processing input
2020-04-12 20:48:48,149 - CAMSA.main      - INFO    - Reading assembly points
2020-04-12 20:48:49,078 - CAMSA.main      - INFO    - Merging assembly points from different sources into a set of unique ones.
2020-04-12 20:48:49,819 - CAMSA.main      - INFO    - Processing assemblies' subgroups
2020-04-12 20:48:49,819 - CAMSA.main      - INFO    - Processing assembly points taking both order and orientation into account
2020-04-12 20:48:49,895 - CAMSA.main      - INFO    - Processing assembly points, taking just order into account
2020-04-12 20:48:50,768 - CAMSA.main      - INFO    - Computing assembly points conflicts
2020-04-12 20:48:55,288 - CAMSA.main      - INFO    - Obtaining a merged assembly, using maximal-matching strategy
Traceback (most recent call last):
  File ".camsa-env/bin/run_camsa.py", line 252, in <module>
    min_cw=args.c_merging_cw_min)
  File "camsa-env/lib/python3.7/site-packages/camsa/core/merging.py", line 131, in maximal_matching
    for cc in networkx.connected_component_subgraphs(G=cover_graph, copy=True):
AttributeError: module 'networkx' has no attribute 'connected_component_subgraphs'
(END)

Thanks

**Comment by aganezov** [comment]
Please advise if you are running conda/PyPi installation of `CAMSA` or the Github source one. There has been a lag in updating the conda /PyPi distribution in my part. I'll be correcting it shortly.

**Comment by navajeet123** [comment]
Hi Aganezov,

This is the Github source one.

Thanks

**Comment by aganezov** [comment]
That's rather strange, as the line 131 (as per you stack trace) in the Github master branch reads 
```python
        for cc in networkx.connected_components(G=cover_graph):
```
and not 
```python
for cc in networkx.connected_component_subgraphs(G=cover_graph, copy=True):
```

Note, sadly, the version numbering does not ideally separate the Github updated version from PyPi/conda one (I'm updating PyPi/conda distributions now to be inline with networkx 2.4 compatibility)

**Comment by navajeet123** [comment]
Dear Aganezov,
many thanks for your response.
Kindly let me know,once you are done with the update. I will install again and would be very much happy to execute once again.

**Comment by aganezov** [comment]
Hello! 
I've created a new version release **1.3** on Github (https://github.com/compbiol/CAMSA/releases/tag/1.3) from the most current master branch snapshot, and I've also updated the PyPi/conda distirbutions with it. 
Most recent Travis CI build (https://travis-ci.org/github/compbiol/CAMSA/builds/674439824) suggests that the release shall work fine, as it successfully ran on OSX/Linux with various vesions of python on several included examples.
Let me know if this addresses your problem!

**Comment by navajeet123** [comment]
Dear Aganezov,
Many thanks!
run_camsa.py completed successfully.


Proceeded with down-line step.
Error:
Traceback (most recent call last):
  File "../camsa-env/bin/camsa_points2fasta.py", line 146, in <module>
    for cc in networkx.connected_component_subgraphs(G=assembly_graph):
AttributeError: module 'networkx' has no attribute 'connected_component_subgraphs'

Please share your views.

Thanks

**Comment by aganezov** [comment]
glad to hear the previous issue was resolved. 

The problem you've outlined seems to be identical to the one mentioned above (i.e., outdated (w.r.t. most recent networkx library) code), but I've checked both the **source code on Github** as well as distributions on **Github release**, **conda**, and **PyPi**, and all have correct updated code, that would not produced such an issue.

Can it be, that while you've updated the `camsa` package to a newwer version, you have not rerun the actual installation process, and thus you have an outdated version of `camsa_points2fasta.py` script in your bin?
